File: content/en/api/embeds/code_snippets/api-embeds-get-all.py
from datadog import initialize, api

# Intialize request parameters including API/APP key
options = {
    'api_key': '<DATADOG_API_KEY>',
    'app_key': '<DATADOG_APPLICATION_KEY>'
}

initialize(**options)

# Call Embed API function
api.Embed.get_all()

File: content/en/api/embeds/code_snippets/api-embeds-get-all.rb
require 'rubygems'
require 'dogapi'

# Initialize API Client
api_key = '<DATADOG_API_KEY>'
app_key = '<DATADOG_APPLICATION_KEY>'

dog = Dogapi::Client.new(api_key, app_key)

# Get all with API Call
dog.get_all_embeds()